apps/web/pages/settings/admin/apps/[category].tsx

```diff
@@ -1,10 +1,11 @@
+import AdminAppsList from "@calcom/features/apps/AdminAppsList";
 import { getAdminLayout as getLayout, Meta } from "@calcom/ui";
 
 function AdminAppsView() {
   return (
     <>
       <Meta title="Apps" description="apps_description" />
-      <h1>App listing</h1>
+      <AdminAppsList baseURL="/settings/admin/apps" className="w-0" />
```

Comment:
`w-0` means we have some flex box magic going on here 👏

Reply:
@sean-brydon yup. Here we want to the child to grow only as much as the parent.